packages/manager/apps/cloud/client/app/dbaas/logs/detail/index/add/logs-index-add.html

```diff
@@ -44,26 +44,20 @@
             ></textarea>
         </oui-field>
 
-        <oui-field data-label="{{ ::'logs_select_options_title' | translate }}">
-            <oui-spinner data-ng-if="ctrl.options.loading"></oui-spinner>
-            <div class="oui-select">
-                <select
-                    class="oui-select__input"
-                    name="logs_index_modal_select"
-                    id="logs_index_modal_select"
-                    data-ng-model="ctrl.index.optionId"
-                    data-ng-options="option.optionId as option.type+', '+option.detail+' - '+option.curNbIndex+'/'+option.maxNbIndex for option in ctrl.options.data"
-                >
-                    <option value=""
-                        >{{::'logs_main_offer' | translate}} -
-                        {{ctrl.quota.data.mainOfferCurrent}}/{{ctrl.quota.data.mainOfferMax}}</option
-                    >
-                </select>
-                <i
-                    class="oui-icon oui-icon-chevron-down"
-                    aria-hidden="true"
-                ></i>
-            </div>
+        <oui-field
+            data-ng-if="!ctrl.isEdit"
```

Comment:
You could remove all `data-` here, because it's a `component` and `data-` are optional when using a component

Reply:
Well it looks that all other fields in the manager also declare `data-` in `oui-field `. Do i have to refactor them all?

Reply:
Could be handled as a separate task since a tons of occurrences exists.